datasets/imagenet2012/README.md

```diff
@@ -0,0 +1,1235 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- other-imagenet-agreement
+multilinguality:
+- monolingual
+paperswithcode_id: imagenet
+pretty_name: ImageNet
+size_categories:
+- 1M<n<10M
+source_datasets:
+- original
+task_categories:
+- image-classification
+task_ids:
+- multi-class-image-classification
+---
+
+# Dataset Card for ImageNet
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** https://image-net.org/index.php
+- **Repository:**
+- **Paper:** https://arxiv.org/abs/1409.0575
+- **Leaderboard:** https://paperswithcode.com/sota/image-classification-on-imagenet
+- **Point of Contact:** mailto: imagenet.help.desk@gmail.com  
+
+### Dataset Summary
+
+ILSVRC 2012, commonly known as 'ImageNet' is an image dataset organized according to the WordNet hierarchy. Each meaningful concept in WordNet, possibly described by multiple words or word phrases, is called a "synonym set" or "synset". There are more than 100,000 synsets in WordNet, majority of them are nouns (80,000+). ImageNet aims to provide on average 1000 images to illustrate each synset. Images of each concept are quality-controlled and human-annotated. In its completion, ImageNet hopes to offer tens of millions of cleanly sorted images for most of the concepts in the WordNet hierarchy. 
```

Comment:
> "There are more than 100,000 synsets in WordNet, majority of them are nouns (80,000+). ImageNet aims to provide on average 1000 images to illustrate each synset." 

This is confusing ImageNet 21k and ImagetNet 1k

Reply:
Thanks for the comment.
> There are more than 100,000 synsets in WordNet, majority of them are nouns (80,000+)
This is more of a generic statement about WordNet and not related to ImageNet directly?

> ImageNet aims to provide on average 1000 images to illustrate each synset.
This is true for ImageNet1k as well.

I think the following line is actually causing more confusion and I will remove it:
> ImageNet hopes to offer tens of millions of cleanly sorted images for most of the concepts in the WordNet hierarchy.